modules/ROOT/pages/_partials/api-table.adoc

```diff
@@ -3,33 +3,37 @@ SDKs for our Mobile Services are available for a variety of platforms, including
 Click on the link:#[✔] to see the API reference documentation:
 
 |===
-| Platform | Core | {keycloak-service} | {unifiedpush-service} | {device-security-service} | {app-security-service}
+| Platform | Core | {keycloak-service} | {unifiedpush-service} | {device-security-service} | {app-security-service} | Data Sync
```

Comment:
@finp - perhaps this should be updated to "Data Sync Client" as it points to the Client API - wdyt?

Reply:
good idea